test/Microsoft.ApplicationInsights.AspNetCore.Tests/SdkVersionTestUtils.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.ApplicationInsights.AspNetCore.Tests
+{
+    public class SdkVersionTestUtils
+    {
+        public static string GetExpectedSdkVersion()
+        {
+            string expectedSdkVersion = "aspnet5";
+
+#if NET451
+        expectedSdkVersion = expectedSdkVersion + "F";
```

Comment:
I thought we are testing it, and any change there would never be caught if we reuse it?